### DISCUSSION TITLE
Sunton 8048S050C: Automatic Dimming Function Not Working

### DISCUSSION THREAD

**Discussion by hibbeldi** [HASwitchPlate/openHASP#775] [Q&A] [answered]
I am experiencing an issue with the automatic dimming functionality on my openHASP setup. While manual dimming and state changes (e.g., setting state to 0 or 1) work correctly, the automatic dimming via idle1 and idle2 does not function (displays stays undimmed all the time). This issue is observed on a specific display model but works fine on a different one (sunton-8048s043c).
What is a little bit odd is that the backlit GPIO is not set for 50c  (GPIO 2 for 43c - GPIO 2 is not available for the 50c)

MQTT Status update works as well (e.g. idle=short, but without dimming).

I already tried factory reset.

Version | 0.7.0-rc12 cd34d89
Environment | sunton-8048s050c_16MB

 "gui": {
        "idle1": 60,
        "idle2": 120,
        "bckl": 2,...}

Any suggestions?
Thank you and kind regards

**Comment by FreeBear-nc** [comment, marked as answer]
Do you have any idle_[short|long|off].cmd files ?
For example, my idle_short.cmd (idle_long.cmd is similar):
```
backlight {"brightness":25}
```
And the idle_off.cmd:
```
backlight {"brightness":255}
```

These files get executed on idle timeouts and when a wake-up event (touch) occurs. For more information, see: https://www.openhasp.com/0.7.0/commands/scripts/#system-scripts

**Reply by hibbeldi** [reply]
Thanks that did the trick! (don't have that on my device with 0.7.0-rc9 b7283c9 and it works there)
--



</body>
</html>
